internal/server/worker_tags_test.go

```diff
@@ -106,3 +109,454 @@ func TestWorkerTags_Create(t *testing.T) {
 		})
 	}
 }
+
+func makeTagsList(tags ...*Tag) []*Tag {
+	var tagsList []*Tag
+	for _, t := range tags {
+		tagsList = append(tagsList, t)
+	}
+	return tagsList
+}
+
+func TestRepository_AddWorkerTags(t *testing.T) {
+	t.Parallel()
+	conn, _ := db.TestSetup(t, "postgres")
+	rw := db.New(conn)
+	wrapper := db.TestWrapper(t)
+	kms := kms.TestKms(t, conn, wrapper)
+
+	assert, require := assert.New(t), require.New(t)
+	repo, err := NewRepository(rw, rw, kms)
+	require.NoError(err)
+	require.NotNil(repo)
+	worker := TestKmsWorker(t, conn, wrapper)
+	// add another worker here to avoid issue with worker version input being locked in at compile time
```

Comment:
I noticed similar "add" tests tend to only have 1 non-error case. Perhaps it has to do with this peculiarity:

When there are more than 1 success cases and these tests are run sequentially, the second success case will not work unless the worker version parameter is incremented by 1. I could hard code a +1 to the version, but then the testcase would not work when run individually.

I tried making a func() { return worker.Version }() as the argument value, but that gets evaluated before the tests are run too.

I could change the comment to "add second worker here to avoid versioning conflicts"

Reply:
I left it as is for now, but I'll probably change it tomorrow since it feels nicer for the testcases to be more linear